apps/native/app/fastlane/Fastfile

```diff
@@ -0,0 +1,312 @@
+# This is the minimum version number required.
+fastlane_version "2.106.0"
+
+default_platform :android
+
+platform :ios do
+
+  # Use the right Xcode version
+  # xcversion(version: "9.4.1")
+
+  before_all do
```

Comment:
unused?

Reply:
yes, it is. Sorry this should not have come for a review right away. More of a draft for now.